test/accessToken.test.js

```diff
@@ -0,0 +1,179 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
- pre commit hook failing in pr check
- update patch version of the sdk so that we can run these tests in cicd as well.